### PR TITLE
[bundle-size][firestore] Refine bundle definition

### PR DESCRIPTION
**Pull request by yifanyang**

### CODE REVIEW

File: repo-scripts/size-analysis/bundle-definitions/firestore-lite.json [hunk 1]
```diff
@@ -66,5 +95,72 @@
...
+        "packageName": "firebase",
+        "versionOrTag": "latest",
+        "imports": [
+          {
+            "path": "firestore/lite",
+            "imports": [
+              "getFirestore",
+              "collection",
+              "doc",
+              "getDoc",
```
Comment (schmidt-sebastian) [inline repo-scripts/size-analysis/bundle-definitions/firestore-lite.json:124]: getDocs?

File: repo-scripts/size-analysis/bundle-definitions/firestore.json [hunk 2]
```diff
@@ -98,5 +127,108 @@
...
+        "packageName": "firebase",
+        "versionOrTag": "latest",
+        "imports": [
+          {
+            "path": "firestore",
+            "imports": [
+              "getFirestore",
+              "collection",
+              "doc",
+              "getDoc",
```
Comment (schmidt-sebastian) [inline repo-scripts/size-analysis/bundle-definitions/firestore.json:156]: getDocs

File: repo-scripts/size-analysis/bundle-definitions/firestore.json [hunk 3]
```diff
@@ -98,5 +127,108 @@
...
+            "path": "firestore",
+            "imports": [
+              "getFirestore",
+              "enableIndexedDbPersistence",
+              "collection",
+              "onSnapshot",
+              "where",
+              "query",
+              "enableNetwork",
+              "disableNetwork"
```
Comment (schmidt-sebastian) [inline repo-scripts/size-analysis/bundle-definitions/firestore.json:196]: I think we should remove all those - they are tested elsewhere.

File: repo-scripts/size-analysis/bundle-definitions/firestore-lite.json (hunk 1 above)
Comment (yifanyang) [reply repo-scripts/size-analysis/bundle-definitions/firestore-lite.json]: Done.

File: repo-scripts/size-analysis/bundle-definitions/firestore.json (hunk 2 above)
Comment (yifanyang) [reply repo-scripts/size-analysis/bundle-definitions/firestore.json]: Done.

File: repo-scripts/size-analysis/bundle-definitions/firestore.json (hunk 3 above)
Comment (yifanyang) [reply repo-scripts/size-analysis/bundle-definitions/firestore.json]: Done.

File: repo-scripts/size-analysis/bundle-definitions/firestore.json [hunk 4]
```diff
@@ -98,5 +127,102 @@
...
+      },
+      {
+        "packageName": "firebase",
+        "versionOrTag": "latest",
+        "imports": [
+          {
+            "path": "firestore",
+            "imports": [
+              "getFirestore",
+              "enableIndexedDbPersistence"
```
Comment (schmidt-sebastian) [inline repo-scripts/size-analysis/bundle-definitions/firestore.json:190]: Can you use `enableMultiTabIndexedDbPersistence`? That might provide more interesting data as it pulls in even more components.

**Review by schmidt-sebastian — APPROVED** [review]

**Review by Feiyang1 — APPROVED** [review]

**Pull request merged**